### PR TITLE
New notebook: Load from numpy

### PR DESCRIPTION
**Pull request by adammichaelwood**
b/123652310

### CODE REVIEW

File: site/en/r2/tutorials/load_data/numpy.ipynb [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "TfRdquslKbO3",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Load Numpy Data with tf.data"
```
Comment (lamberta) [inline site/en/r2/tutorials/load_data/numpy.ipynb]: Looks like it should be "NumPy" (here and throughout)

File: site/en/r2/tutorials/load_data/numpy.ipynb [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-0tqX8qkXZEj",
+        "colab_type": "text"
+      },
+      "source": [
+        "This tutorial provides an example of loading data from Numpy arrays into a `tf.data.Dataset`.\n",
+        "\n",
+        "This examples loads the MNIST dataset from a `.npz` file. However, the source of the Numpy arrays is not important.\n"
```
Comment (lamberta) [inline site/en/r2/tutorials/load_data/numpy.ipynb]: examples => example

File: site/en/r2/tutorials/load_data/numpy.ipynb [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0dvl1uUukc4K",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Prepare the datasets\n",
+        "\n",
+        "For this examples, we're going to flatten the input data, shuffle it, and batch it."
```
Comment (lamberta) [inline site/en/r2/tutorials/load_data/numpy.ipynb]: Prefer "you" instead if "we"
https://developers.google.com/style/person

File: site/en/r2/tutorials/load_data/numpy.ipynb [hunk 4]
```diff
@@ -0,0 +1,311 @@
+{
```
Comment (yashk2810) [inline site/en/r2/tutorials/load_data/numpy.ipynb]: <p>Why not use tf.keras.dataset.mnist.load_data()?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/numpy.ipynb]: <p>Please no functions within functions.</p><p><br></p><p>Also, </p><p><br></p><p>`return tf.reshape(x, [784]) / 255.`</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/numpy.ipynb]: <p>Why not use Sequential API?</p><p><br></p><p>This is straightforward, so sequential makes more sense here.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (lamberta) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: I think the point is to explicitly show loading a .npz file

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: Not, as I understand it, `.npz` specifically - but in-scope NumPy arrays.

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: > no functions in functions
okay

> `return tf.reshape(x, [784]) / 255.`

this causes a type error:

```
TypeError: x and y must have the same dtype, got tf.uint8 != tf.float32
```

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: I didn't write it, so I can't answer why. 
But I don't mind changing it.

File: site/en/r2/tutorials/load_data/numpy.ipynb (hunk 1 above)
Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: fixed

File: site/en/r2/tutorials/load_data/numpy.ipynb (hunk 2 above)
Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: fixed

File: site/en/r2/tutorials/load_data/numpy.ipynb (hunk 3 above)
Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: fixed

File: site/en/r2/tutorials/load_data/numpy.ipynb (hunk 4 above)
Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: Removed altogether.

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/numpy.ipynb]: Done.

**Comment by adammichaelwood** [comment]
Simplified. Maybe it's good now.

**Comment by yashk2810** [comment]
This looks nice! Thanks

**Comment by lamberta** [comment]
Looks good. Thanks, Adam

**Pull request merged**